src/header/utils.js

```diff
@@ -66,7 +66,12 @@ export const getToolsMenuItems = ({ studioBaseUrl, courseId, intl }) => ([
   {
     href: `${studioBaseUrl}/export/${courseId}`,
     title: intl.formatMessage(messages['header.links.export']),
-  }, {
+  },
+  {
+    href: `${studioBaseUrl}/api/content_tagging/v1/object_tags/${courseId}/export/`,
+    title: intl.formatMessage(messages['header.links.exportTags']),
+  },
```

Comment:
Inclusion of this menu item needs to be gated by the [same flags that we gate the other taxonomy menu items](https://github.com/open-craft/frontend-app-course-authoring/blob/rpenido/fal-3610-add-export-course-tags-menu/src/index.jsx#L56-L71), which also means it needs tests added [like these](https://github.com/open-craft/frontend-app-course-authoring/blob/rpenido/fal-3610-add-export-course-tags-menu/src/header/utils.test.js).